src/main/java/org/springframework/integration/dsl/amqp/AmqpOutboundEndpointSpec.java

```diff
@@ -33,9 +33,6 @@
 		this.target = new AmqpOutboundEndpoint(amqpTemplate);
 		this.target.setExpectReply(expectReply);
 		this.target.setHeaderMapper(this.headerMapper);
-		if (expectReply) {
-			this.target.setRequiresReply(true);
-		}
```

Comment:
I don't understand these changes - 

> don't make sense because of public `ConsumerEndpointSpec.requiresReply()`

Even though there's a public setter, surely we should default this to true for gateways?